src/test/files/DevonUcImplSecurityConstraintCheckRolesAllowedCase.java

```diff
@@ -0,0 +1,11 @@
+package com.devonfw.testapplication.testcomponent.logic.impl;
+
+public class TestClassImpl implements TestClass {
+
+    @Override
+    @RolesAllowed(ApplicationAccessControlConstants.PERMISSION_TEST)
```

Comment:
Please change to to be compliant with `devonfw` (`Config` instead of `Constants`):
`@RolesAllowed(ApplicationAccessControlConfig.PERMISSION_SAVE_ENTITY)`